detections/endpoint/cisco_isovalent___suspicious_image_use.yml

```
@@ -0,0 +1,79 @@
+name: Cisco Isovalent - Suspicious Image Use
```

Perhaps we should try and avoid the use of the word suspicious here. I like the word you used `Non Allowlisted` so let's use that. That way we can create a better picture for the end-user.

```suggestion
name: Cisco Isovalent - Non Allowlisted Image Use
```